build.gradle

```diff
@@ -285,7 +285,9 @@ subprojects { subproj ->
 			}
 		}
 
-		def signArtifacts = !project.version.contains('SNAPSHOT')
+		def isSnapshot = project.version.contains('SNAPSHOT')
+		def isContinuousIntegrationEnvironment = Boolean.parseBoolean(System.getenv("JITPACK"))
+		def signArtifacts = !(isSnapshot || isContinuousIntegrationEnvironment)
```

Comment:
Didn't you want to sign the artifacts for Jitpack?

Reply:
It fails on JitPack as there are no credentials available over there.

The purpose, namely depending on a **branch** SNAPSHOT build, is served well w/o signing and usually, one could use our official **master** SNAPSHOT distro anyway.

Reply:
Ok, makes sense, sorry I misremembered our conversation about signing.